spec/mailers/digest_mailer_spec.rb

```diff
@@ -3,19 +3,22 @@
 RSpec.describe DigestMailer, type: :mailer do
   let(:user) { create(:user) }
   let(:article) { build_stubbed(:article) }
+  let(:from_email_address) { "custom_noreply@forem.com" }
 
   describe "#digest_email" do
     before do
       allow(article).to receive(:title).and_return("test title")
+      allow(Settings::SMTP).to receive(:provided_minimum_settings?).and_return(true)
+      allow(Settings::SMTP).to receive(:from_email_address).and_return(from_email_address)
     end
 
     it "works correctly" do
```

Comment:
Quick, non-blocking suggestion: could we please add `:aggregate_failures` here since there are a handful of expectations that could fail within this one block?
```suggestion
    it "works correctly", :aggregate_failures do
```